page/about-jquery/using-jquery.md

```diff
@@ -0,0 +1,155 @@
+      <li><a href="http://github.com">Products</a></li>
+    </ul>
+  </nav>
+  <script src="jquery.js"></script>
+  <script src="demo.js"></script>
+  <script src="main.css"></script>
+</body>
+</html>
+```
+
+The `<script src>` tags reference the jQuery library, which you should have downloaded and copied into your directory, as well as the files `demo.js` and `main.css`. If you don't already have them, create these two files in the same directory and copy their content as shown below.
+
+* **Note:** Normally, these different types of files are sorted into folders that represent the overall structure of the webpage, reflecting the separation between content, action, and style, or, in parallel terms, model, controller, and view. Though you could alter the CSS of an element directly in the JavaScript by using jQuery's [`.css()`](http://api.jquery.com/css/) method, that piece of information about the style won't be included with the rest of the CSS, making it hard to find in the future, or for other developers. Similarly, you could construct HTML and text in a JavaScript file to be appended to the DOM, but again, that content won't be included with the rest of the HTML, where it belongs. Instead, organize and label your content in HTML, use JavaScript and jQuery to assign events to the elements, and let the CSS take care of the styling.
+
+```
+// demo.js
+$( document ).ready(function( event ) {
+
+  // Your code here
+
+  event.preventDefault();
+
+});
+```
+
+```
+// main.css
+a.active {
+  font-weight: bold;
+}
+```
+
+**Important:** *You must place the remaining jQuery examples inside the `ready` event so that your code executes when the document is ready to be worked on, as explained in the [previous setup tutorial](http://learn.jquery.com/about-jquery/how-to-setup-jQuery/).*
+
+Add the [.addClass()](http://api.jquery.com/addClass/) call inside the `ready` event:
+
+```
+$( "a" ).addClass( "active" );
+```
+
+All `<a>` elements are now bold. If you right click on a link and select "Inspect Element" you'll see that the class of "active" has been added to all `<a>` elements.
+
+To remove an existing class, use [.removeClass()](http://api.jquery.com/removeClass/):
+
+```
+$( "a" ).removeClass( "active" );
+```
+
+Let's bring this all together now. What if we only want the most recently clicked link to turn bold? jQuery can alter groups of elements by selecting all of them, but it can also point to a specific element using the `$(this)` referant.
+
+```
+var $links = $( "a" ).on( "click", function makeActive( event ) {
+
+  $links.removeClass( "active" );
+  $( this ).addClass( "active" );
+
+  event.preventDefault();
+
+});
+```
+
+First, we save the selection of all `<a>` elements to the variable `$links`, so we can reuse the selection later without having to query the DOM again.
+
+Next, we remove the "active" class from all `<a>` elements, guaranteeing that the link clicked previously is no longer bold or "active".
```

Comment:
I'm going to get a little pedantic about the order of operations here.

1. search the document for all `<a>` elements.
2. attach a click event handler to the selection
3. save the collection to a variable called $links
4. click on a link
5. remove the active class from the $links selection we saved earlier
6. add the class to the link we clicked on
7. prevent the default action (navigation)

The order of 1-3 seems >kind< of important to me